src/main/java/org/wildfly/security/ssl/SSLUtils.java

```diff
@@ -127,12 +127,13 @@ private SSLUtils() {}
                             } catch (NoSuchAlgorithmException ignored) {}
                         }
                 }
-                throw ElytronMessages.log.noAlgorithmForSslProtocol();
```

Comment:
I think we should still throw this here but add trace logging in the ```catch``` body above to indicate the ```protocol``` and ```provider``` pair that resulted in the ```NoSuchAlgorithmException```.

Reply:
Fixed